packages/expo-age-range/src/AgeRange.web.ts

```diff
@@ -0,0 +1,5 @@
+import { AgeRangeRequest, AgeRangeResponse } from './ExpoAgeRange.types';
+
+export async function requestAgeRangeAsync(_: AgeRangeRequest): Promise<AgeRangeResponse> {
+  throw new Error('AgeRange module is not supported on web platform.');
```

Comment:
Same here, maybe we can just return a full range? I guess web browsers might implement it at some point, so it would be great if people don't have to do platform checks.